template/app/src/main/java/com/xmartlabs/template/ui/login/LoginFragment.java

```diff
@@ -0,0 +1,68 @@
+package com.xmartlabs.template.ui.login;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ProgressBar;
+
+import com.hannesdorfmann.fragmentargs.annotation.FragmentWithArgs;
+import com.xmartlabs.template.R;
+import com.xmartlabs.template.TemplateApplication;
+import com.xmartlabs.template.ui.Henson;
+import com.xmartlabs.template.ui.common.TemplateFragment;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+
+@FragmentWithArgs
+public class LoginFragment extends TemplateFragment<LoginView, LoginPresenter> implements LoginView {
+  @BindView(R.id.log_in_button)
+  Button loginButton;
+  @BindView(R.id.progress_bar)
+  ProgressBar progressBar;
+
+  @NonNull
+  private final LoginPresenter presenter = LoginPresenter.builder().build();
```

Comment:
It's as we do with fragments without args.
For the sake of consistency, we should do it the same way here i think